products/healthcare/api.yaml

```diff
@@ -424,9 +424,25 @@ objects:
               - parser_config.0.allow_null_header
               - parser_config.0.segment_terminator
               - parser_config.0.schema
+              - parser_config.0.version
             description: |
               JSON encoded string for schemas used to parse messages in this
               store if schematized parsing is desired.
+            # conflicts:
```

Comment:
@danawillow when I uncomment this out I get the following error:

`RuntimeError: version does not exist`

I also tried `parser_config.0.version`.

This seems to be consistent with other uses of conflicts. Do you know what I'm missing?

Reply:
I found a comment somewhere else that made it sound like we don't have support for conflicts on nested fields. I don't think it would be super difficult to add, but our team isn't adding new MMv1 features at the moment. I'd recommend just leaving it out, unless you want to dig through the generator code to make the fix (we _do_ have it for at_least_one_of, but I think conflicts has some nicer things like checking that the referenced property actually exists so we would need to merge their implementations)

Reply:
Sounds good, I just removed it for now.